src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -55,25 +55,13 @@
   </ItemGroup>
```

Comment:
so how does this end up working?  does the MSBuild libs we reference automatically locate properly?

Reply:
We invoke MSBuild locator in the BulidHost processes and just do the right thing in that case. (Or nearly right thing -- there are some edge cases we might not support, but nobody else does those either.)